### PR TITLE
Map Pipes to their sub expressions

### PR DESCRIPTION
**Pull request by evanfuture**
Here is my naive fix for #109. Naive because I'm not sure of what edge cases there may be.  Test html was updated, and passes.

### CODE REVIEW

**Comment by shaharkazaz** [comment]
@evanfuture Thanks for the PR! Let's make this a bit more generic since your solution won't support pipe chaining:
```html
{{ t('last "one"') | json | pipe2 }}
```

Instead of mapping the expression once, in case of binding pipes loop until it's not a pipe anymore.

**Comment by evanfuture** [comment]
Good point.  I pushed that change just now.

File: src/keys-builder/template/structural-directive.extractor.ts [hunk 1]
```diff
@@ -73,8 +74,13 @@ export function traverse(
   }
 }
 
+function unwrapExpression(exp: AST): AST {
```
Comment (shaharkazaz) [inline src/keys-builder/template/structural-directive.extractor.ts:77]: Great! Can you please add the following comment:
```ts
        /*
        * Handle pipes in interpolation
        *
        * @example
        * {{ t('another(test)') | pipe1 | pipe2 }} will return the `t('another(test)')` node
        * */
```

**Comment by shaharkazaz** [comment]
@evanfuture I see that your branch isn't from the master branch.
Open a new PR with these fixes from branch master to branch master and follow the commit message structure (just run `npm run commit` after adding the files to git)

**Comment by evanfuture** [comment]
Moved to this PR: https://github.com/ngneat/transloco-keys-manager/pull/111
